tasks/cops_documentation.rake

```diff
@@ -25,9 +25,10 @@ task generate_cops_documentation: :yard_for_generate_documentation do
   end
 
   def examples(examples_object)
-    content = h3('Example')
-    content += examples_object.map { |e| code_example(e) }.join
-    content
+    examples_object.each_with_object(''.dup) do |example, content|
+      content << h3(example.name == '' ? 'Example' : "Example #{example.name}")
```

Comment:
Maybe `Example (\`...\`)` or `Example for \`...\`` would read better here (just looking at the generated manual.

Reply:
And we should probably add a heading `Examples` when there are multiple code examples for one cop.

Reply:
Like YARD, I added `Examples` uniformly. This is also to unify the `H3` header with `Examples`.
For the example name new `H4` header is provided.

Perhaps it may be slightly different from the image 😟  , but this format seems not to be bad.
https://github.com/koic/rubocop/blob/ad203f8dc6baac7fb80f210cc9b2f1346f116ffc/manual/cops_style.md

WDYT?